core/README.md

```diff
@@ -33,3 +33,31 @@ The process of adding new collaterals depends on the token type used. This is du
+    2. Read the contract and determine the slot address of the variable:
+       - Generally a slot number can be determined by counting definition of variables in the contract source code, but there are exceptions, [please read the docs on the solidity version the contract was compiled with](https://docs.soliditylang.org/en/v0.8.13/internals/layout_in_storage.html)
+       - Experimenting with blockchain fork (e.g. hardhat) helps: try to fetch the value you're looking for / overwrite it / ... and validate that it's correct via some public method or comparing against your expectation. See section [Overwriting values of price oracles](./README.md#overwriting-values-of-price-oracles)
+    3. Extend collateral config with the proper slot addresses.
+    4. If needed, add the oracle type to `types` file if the existing types are not sufficient to cover for the set of values you need.
+
+### Overwriting values of price oracles
+
+Beforehand: count the variables in contract from top to bottom of the contract (__not__ the file). Skip `struct` defenitions. The order number of your variable is __approximately__ the slot address of it.
+
+1. The main function that is used in the process is [`hardhat_setStorageAt`](https://hardhat.org/hardhat-network/docs/reference#hardhat_setstorageat). You can use it to overwrite values. But for that you have to have the fork running:
+    - `npm run hardhat`
+2. Find the way to validate the price. Usually it's possible to read it via some method. For example contract https://etherscan.io/address/0xFe7a2aC0B945f12089aEEB6eCebf4F384D9f043F#code has `peek` function that returns the current price.
+3. Ensure that you're authorized to do call your validation function (or the function is publicly available and can be called by anyone)
+  3.1. If you're not authorized - you have to overwrite the mapping that stores the authorizations before reading or calling any of the functions. The process is no different from overwriting any other value.
+4. Overwrite the value on the running hardhat fork, write your script that does the following:
+   - get the provider object. (e.g. call `getProvider` from `provider.ts`)
+   - determine the contract name (or add it). Use `contract.ts` file.
+   - Call the overwrite function, most likely `/helpers/hardhat` package might be of help here since it already has some funcitonality for overwrites defined.
+   - validate that the overwrite was successful:
+       - if you were whitelisting yourself (giving auth to your wallet) - call a function of the contract that requires this auth:
+           - `overwriteUIntMapping`
+           - `overwriteUIntValue`
+       - if you are whitelisted, call the function of the contract (`await contract.function_name(...params)`) that returns the price. Compare it with the value you have submitted.
```

Comment:
This part seems like you're explaining how to write a (test) script. But it's not specific enough. Eg `determine the contract name (or add it). Use contract.ts file` – which contract name, how to determine it?

Can you please instead provide the script(s)? The script that will ask collateral name via prompt, print its OSM address as a url to etherscan and the instruction to count something specific, then wait for the count, then ask for something else, eg method name, validate it, and finally print the oracle config. Otherwise, I am afraid we will keep iterating this readme indefinitely. As [most of the questions asked previously](https://github.com/sidestream-tech/unified-auctions-ui/pull/457#discussion_r974307134) still not answered:
- What `private variables` need to be extracted? Do you have their names, their types, other ways of understanding that?
- What to do with the slot address after it was determined? In which exact variable to put it, in which format?
- What is the difference between `CurrentAndNextPrice` and the other type? How to determine which one is which?
- What is `hasDelay` value and how to determine it?
- Where is the example of the config that needs to be added?
- What is `offset`? How to determine it?
- What is `slotPriceValueBeginsAtPosition`? How to determine it?